### ISSUE TITLE
Blackfire can't be installed on php:8.1-cli-alpine

### ISSUE THREAD

**Issue by spaceemotion** [mlocati/docker-php-extension-installer#506]
According to [their article](https://blog.blackfire.io/php-8-1-support.html), blackfire should support php8.1 by now. I have tried to install it in a `php:8.1-cli-alpine` container, but no such luck.

Is there a different reason that prevents this package from being able to run on php8.1?

Thanks in advance!

**Comment by mlocati** [comment]
> I have tried to install it in a `php:8.1-cli-alpine` container, but no such luck.

Could you be more precise? What's the error you are experiencing?

**Comment by spaceemotion** [comment]
The `cli` container seems to work fine today, might have been a cache issue yesterday (I got extension load errors whenever I did anything; composer, phpunit, ...).

I installed `blackfire` in a php:8.1-fpm-alpine container and got no errors during build. However, when I open up a webpage, the connected nginx returns with "502 bad gateway". There is no log output on why that is the case though. (and yes, `catch_workers_output` is enabled).

After doing a couple rebuilds, the problem went away. Not sure why I got both problems initially......
